intranet/static/js/responsive.core.js

```diff
@@ -1,30 +1,33 @@
 /* global $ */
-
 function toggleNavbar() {
     $n = $(".main > .nav").eq(0);
     $g = $(".nav-g");
     // get css left, remove px ending if it exists, and check if 0 (-202px if hidden)
-    if($n.css('left').split(/[^\-\d]+/)[0] == 0) { // hide
-        $n.animate({ left: "-202px" }, 200);
+    if ($n.css('left').split(/[^\-\d]+/)[0] === 0) { // hide
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$n" is not defined.](https://www.codacy.com/app/tjhsst-ion/ion/file/2747525205/issues/source?bid=3212528&fileBranchId=3355681#l6)